src/main/java/com/agiletec/plugins/jacms/aps/system/services/resource/model/ImageResource.java

```diff
@@ -132,7 +132,13 @@ public void saveResourceInstances(ResourceDataBean bean, List<String> ignoreMeta
                 ((BaseResourceDataBean)bean).setFile(tempMasterFile);
             }
 
-            setMetadata(getImgMetadata(tempMasterFile, ignoreMetadataKeys));
+            Map<String, String> imgMetadata = new HashMap<>();
+
+            if(!bean.getMimeType().contains("svg")) {
```

Comment:
Better using `"image/svg"` as done in jsp

Reply:
Done